.gitignore

```diff
@@ -22,3 +22,4 @@ tags
 # Locale files
 locale/*/*.edit.po
 locale/*/*.po.time_stamp
+.tramp_history
```

Comment:
@tstrachota Let me know if this can't be added to `.gitignore`

Reply:
This is totally fine. Just move it below the comment "#Editors" so that people roughly know what it belongs to.